Sources/Components/Fiks Ferdig/FiksFerdigInfoView/FiksFerdigShippingInfoViewModel.swift

```diff
@@ -3,17 +3,20 @@ import UIKit
 public class FiksFerdigShippingInfoViewModel {
     public let providers: [FiksFerdigShippingInfoCellViewModel]
     public let headerViewModel: FiksFerdigAccordionViewModel
+    public let noProviderText: String?
```

Comment:
Since having no providers is a valid state, maybe this text shouldn't be optional?

Reply:
Good point. I made it optional so as to not turn this into a breaking change. What's your opinion on this? @BjornRuud @AtishPapa

Reply:
It won't actually break anything since older clients aren't using it. Or am I missing something here?

Reply:
Right. I'll change this then.